src/react/experimental/transition-manager.js

```diff
@@ -0,0 +1,209 @@
+
+/* eslint-disable max-depth */
```

Comment:
Is this actually needed? Could it be used to wrap only the part that is needed?

Reply:
Moved to code block where it is needed.